<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBConfigDialog</class>
 <widget class="QDialog" name="DBConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="leAddress">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblAddress">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adres serwera</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePort">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblPort">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nr portu</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblUser">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Użytkownik:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUser">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblPassword">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hasło:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDatabase">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblDatabase">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Baza danych:</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>171</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>105</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(0,190,0);
}

QPushButton{
color: white;
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(0,190,0), stop: 0.7 rgb(0,150,0));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(0,150,0);
}
</string>
   </property>
   <property name="text">
    <string>Uruchom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>105</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(255,102,102);
}

QPushButton{
color: white;
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(255,102,102), stop: 0.7 rgb(255,50,50));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(255,50,50);
}
</string>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <zorder>leAddress</zorder>
  <zorder>lblAddress</zorder>
  <zorder>lePort</zorder>
  <zorder>lblPort</zorder>
  <zorder>lblUser</zorder>
  <zorder>leUser</zorder>
  <zorder>lePassword</zorder>
  <zorder>lblPassword</zorder>
  <zorder>leDatabase</zorder>
  <zorder>lblDatabase</zorder>
  <zorder>runButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>line_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
